developer_manual/app/appenv.rst

```diff
@@ -0,0 +1,77 @@
+================================
+App development setup via Docker
+================================
+
+.. sectionauthor:: Wolfgang Popp <mail@wolfgang-popp.de>
+
+This page describes a simplified development environment for app development via docker.
+The docker image is pre-configured with build instructions for some existing apps.
+If you want to create a new app you could base it on the `vueexample <https://github.com/skjnldsv/vueexample.git>`_ app, which is also supported by the container.
+In case you want to work on an app that is not supported by the container by default you will have to go through an extra step (see :ref:`custom-build-commands`).
+
+Setup Docker and Git
+--------------------
+
+First you need to set up Docker, `Docker Compose <https://docs.docker.com/compose/install/>`_ and Git.
+
+* Ubuntu: `docker <https://docs.docker.com/install/linux/docker-ce/ubuntu/>`__, Git: ``sudo apt install git``
+* macOS: `docker <https://docs.docker.com/docker-for-mac/install/>`__, Git: ``brew install git``
+* Windows: `docker <https://docs.docker.com/docker-for-windows/install/>`__, `Git <https://git-scm.com/download/win>`_
+
+Setting up the docker container
+-------------------------------
+
+The following instructions will set up the app development environment and the `vueexample <https://github.com/skjnldsv/vueexample.git>`_ app.
+For other apps simply clone a different repo into the apps folder.
+See the comments in the code block below.
+Of course, multiple apps can be installed at once.
+
+.. code-block:: bash
+
+    git clone https://github.com/David-Development/nextcloud-dev-docker.git
```

Comment:
I would actually prefer to just add things to the documentation which are handled in the Nextcloud repos. How about adding a development variant to the official docker image, that can then be used. We can then add an example of the docker-compose file to this documentation. That would also have the benefit, that not every user needs to build the image themselve.

Reply:
I agree. @tilosp what do you think about adding a dev variant?

Reply:
yes i agree too. But we should put the dev image in a different docker hub repo (for example `nextcloud/dev`)
if @David-Development is fine with it we could simply move the repo to the nextcloud org

Reply:
@tilosp Yes, for sure! 👍

Reply:
Still up for this? :)

Reply:
Can anyone bring this PR up to date? I'm fine with merging it into master.

Reply:
@David-Development give me access to your repo and I'll move it onto /nextcloud if you're still ok with it :)
It could hopefully attract maintainers as well :wink:

Reply:
@David-Development I'm am now a collaborator, but not an owner of your repo, I cannot move it :)

Reply:
> @David-Development I'm am now a collaborator, but not an owner of your repo, I cannot move it :)

You can't get admin of a personal repo on Github 🙈

Reply:
The workaround would be to create a new organization that you both are admins of and @David-Development transfers the repo to that org. Afterwards @skjnldsv can transfer it to the Nextcloud org 🙃

Reply:
xD!
Sorry for the confusion David! :grin: 
Thanks for the help you two!!

Reply:
@skjnldsv Just transferred the repo to you 😉